systemd/systemd.go

```diff
@@ -941,24 +947,34 @@ loop:
 		case <-giveup.C:
 			break loop
 		case <-check.C:
-			bs, err := s.systemctl("show", "--property=ActiveState", serviceName)
-			if err != nil {
-				return err
+			allStopped := true
+			stillRunningServices := []string{}
+		serviceCheck:
+			for i := 0; i < len(serviceNames); i++ {
+				bs, err := s.systemctl("show", "--property=ActiveState", serviceNames[i])
+				if err != nil {
+					return err
+				}
+				if !isStopDone(bs) {
+					stillRunningServices = append(stillRunningServices, serviceNames[i])
```

Comment:
nitpick: this could use a range instead. I find the resulting code simpler but it's subjective so feel free to ignore if you disagree
```suggestion
			for _, service := range serviceNames {
				bs, err := s.systemctl("show", "--property=ActiveState", service)
				if err != nil {
					return err
				}
				if !isStopDone(bs) {
					stillRunningServices = append(stillRunningServices, service)
```